samples/MASMessagingSample/src/main/java/com/ca/mas/masmessagingsample/activity/GroupDetailActivity.java

```diff
@@ -88,7 +105,12 @@ public boolean onOptionsItemSelected(MenuItem item) {
             //
             // http://developer.android.com/design/patterns/navigation.html#up-vs-back
             //
-            NavUtils.navigateUpTo(this, new Intent(this, GroupListActivity.class));
+            Class<?> clazz = GroupListActivity.class;
```

Comment:
Just an old legacy way to mention refer class objects.. you can see some old java legacy systems have such kind of nomenclatures.. i got this habit since then.. shouldn't be a concern. You can see this link and the code posted there... https://stackoverflow.com/questions/9886266/is-there-a-way-to-instantiate-a-class-by-name-in-java